projects/exui-common-lib/src/lib/gov-ui/components/gov-uk-date/gov-uk-date.component.spec.ts

```diff
@@ -49,4 +49,35 @@ describe('GovUkDateComponent', () => {
     expect(fieldsetElement.properties.config.legend).toEqual('Date field');
     expect(fieldsetElement.properties.config.hint).toEqual('This is a hint');
   });
+
+  it('should pass validation for a valid date', () => {
+    const dateValidator = component.DateValidator();
+    component.formGroup = new FormGroup({
+      id_day: new FormControl('', dateValidator),
```

Comment:
Info only: in the test dateValidator only needs applying to the first field as the method checks dd mm yyyy anyway.